### PR TITLE
ISPN-5251 Near cache size must be provided

### PR DESCRIPTION
**Pull request by galderz**
https://issues.jboss.org/browse/ISPN-5251

### CODE REVIEW

File: documentation/src/main/asciidoc/user_guide/chapter-48-Using_Hot_Rod_Server.adoc [hunk 1]
```diff
@@ -590,6 +590,16 @@ executed, the value is directly fetched from the near cache. The downside of
 eager mode is that it's a more heavyweight mode and hence takes up more
 resources in terms of bandwith.
 
+* Size: When near cache is enabled, its size must be configured by defining
+the maximum number of entries to keep in the near cache. When the maximum is
+reached, near cached entries are evicted using a least-recently-used (LRU)
```
Comment (wburns) [inline documentation/src/main/asciidoc/user_guide/chapter-48-Using_Hot_Rod_Server.adoc]: Would we want to use LIRS instead ?

Comment (galderz) [inline documentation/src/main/asciidoc/user_guide/chapter-48-Using_Hot_Rod_Server.adoc]: I created [ISPN-5320](https://issues.jboss.org/browse/ISPN-5320) a few days back. Once we have that in, we should be able to switch between LRU and LIRS

Comment (wburns) [inline documentation/src/main/asciidoc/user_guide/chapter-48-Using_Hot_Rod_Server.adoc]: :+1: Oh, okay great

**Comment by wburns** [comment]
Pulling...

**Comment by wburns** [comment]
Integrated into master, Thanks @galderz !

**Pull request closed without merging**